recipes/ozz-animation/all/conanfile.py

```diff
@@ -1,66 +1,204 @@
 import os
-from conans import ConanFile, CMake, tools
+from pathlib import Path
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, rename, get, replace_in_file
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
 
 class OzzAnimationConan(ConanFile):
     name = "ozz-animation"
     description = "Open source c++ skeletal animation library and toolset."
     license = "MIT"
-    topics = ("conan", "ozz", "animation", "skeletal")
+    topics = ("ozz", "animation", "skeletal")
     homepage = "https://github.com/guillaumeblanc/ozz-animation"
     url = "https://github.com/conan-io/conan-center-index"
-    exports_sources = ["CMakeLists.txt"]
-    generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
     options = {
         "fPIC": [True, False],
+        "shared": [True, False],
+
+        #runtime geometry library (skinning jobs)
+        "ozz_geometry": [True, False],
+
+        #runtime animation library
+        "ozz_animation": [True, False],
+
+        #library for parsing CLI args
+        "ozz_options": [True, False],
+
+        #library for offline processing
+        "ozz_animation_offline": [True, False],
+
+        #library for creating tools
+        "ozz_animation_tools": [True, False],
+
+        #gltf2ozz CLI tool
+        "tools": [True, False],
     }
     default_options = {
         "fPIC": True,
+        "shared": False,
+        "tools": False,
+        "ozz_geometry": True,
+        "ozz_animation": True,
+        "ozz_options": False,
+        "ozz_animation_offline": False,
+        "ozz_animation_tools": False,
     }
-    short_paths = True
 
-    _source_subfolder = "source_subfolder"
-    _build_subfolder = "build_subfolder"
+    def validate(self):
+        def _ensure_enabled(opt, req):
+            if getattr(self.options, opt):
```

Comment:
```suggestion
            if self.options.get_safe(opt):
```
Just a suggestion. `getattr()` is ok, too.